src/scenekit.cs

```diff
@@ -2720,6 +2720,10 @@ interface SCNSceneSourceLoading {
 		[Field ("SCNSceneSourceStrictConformanceKey")]
 		NSString StrictConformanceKey { get; }
 		
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "No longer supported.")]
+		[Deprecated (PlatformName.TvOS, 11, 0, message: "No longer supported.")]
+		[Deprecated (PlatformName.iOS, 11, 0, message: "No longer supported.")]
+		[Deprecated (PlatformName.MacOSX, 10, 13, message: "No longer supported.")]
```

Comment:
There's no need for a custom message. The IDE can handle this with a default, localized, message.

Reply:
Oh that's a good point, it can indeed!